test/amqp-integration-test.py

```diff
@@ -85,8 +87,17 @@ def run_node_test():
         print("\nIssuing failed")
         die(ExitStatus.NodeFailure)
 
+    # As OCSP-Updater is generating responses indepedantly of the CA we sit in a loop
+    # checking OCSP until we either see a good response or we timeout (5s).
     ee_ocsp_url = "http://localhost:4002"
     issuer_ocsp_url = "http://localhost:4003"
+    timeout = time.time() + 5
```

Comment:
Also, you'll need this same sort of loop for verify_ocsp_revoked once you move the revocation OCSP update out of the CA, so you may as well break it out into a helper function now.